### DISCUSSION TITLE
Support of multiple codepages

### DISCUSSION THREAD

**Discussion by BraINstinct0** [winfsp/winfsp#389] [Q&A]
Will using different codepages(cp949, UTF-8, EUC-KR) cause any issues?

**Comment by BraINstinct0** [comment]
To be specific, this is a question regarding whether WinFSP has modules like iconv packed in.

**Reply by billziss-gh** [reply]
WinFsp does not know anything about codepages. It uses Unicode internally.

**Reply by BraINstinct0** [reply]
Thanks. Then encodings/code pages can safely be handled by either the system or when data is passed to WinFSP

**Reply by billziss-gh** [reply]
I believe that is the case, yes.
